frontend/common/src/components/form/Select/SelectFieldV2.tsx

```diff
@@ -88,6 +95,23 @@ const LocalizedNoOptionsMessage = <
     </components.NoOptionsMessage>
   );
 };
+
+const StateStyledSelectContainer = ({
+  children,
+  ...props
+}: ContainerProps<Option | Group<Option>>) => {
+  // @ts-ignore
```

Comment:
I think eslint wants me to use something more specific here but I'm not sure what that would be.

Reply:
I was able to find some documentation on this but it doesn't seem to work 😢 

https://react-select.com/typescript#custom-select-props

Reply:
Not sure if this is useful, but found an [issue](https://github.com/JedWatson/react-select/issues/4804) referencing custom select props and has more sample code than the docs.

Reply:
That seems to have worked. Thanks!